tests/test_MocLogin.py

```diff
@@ -0,0 +1,101 @@
+from django.test import TestCase
+from core.models import AtmosphereUser as User
+from core.models import Provider
+from core.models import PlatformType
+from core.models import ProviderType
+
+from threepio import logger
+
+from atmosphere import settings
+
+
+# This is assuming a clean database
+class MocLogin(TestCase):
+    username = test_settings['username']
```

Comment:
OK, what doesn't make sense is that the UserFactory is a pattern that will create users in atmosphere regardless whether or not they exist in openstack.  This is not enabling us to test what we want to test.  Cyverse will use the UserFactory as they are using in lieu of running the add user script, which, based on our use case, we don't want to do.

Furthermore we are not testing logging into openstack.  Basically, in order to test within CI, I need to simulate/mock up a keystone connection.  

Likewise Cyverse will use the ProviderFactory in lieu of running the add provider script, which is another thing we don't want to do.